packages/cli-zendesk-ticket/src/cli.ts

```diff
@@ -75,32 +84,36 @@ class CLI {
 
   private formData?: FormData
 
-  constructor () {
+  constructor() {
     this.dbg = debug(`webhooks-zendesk-ticket`)
   }
 
-  saveTicket = async (ticket: Ticket) => {
+  saveTicket = async (ticket: Ticket, tries: number, counter: number) => {
+    if (counter >= tries) {
+      console.log(`Limite máximo de tentativas alcançado para o ${ticket.id}`)
+      return
+    }
     try {
-      await saveTickets(ticket)
+      await saveTicket(ticket)
```

Comment:
@rolivegab uma dica apenas, eu acho mais fáćil de ler quando fica claro de onde vem as coisas, esse metódo `saveTicket` se confunde com um método `this.saveTicket` o uso do `this` localiza o acesso ao atributo mas se você coloca nomes mais claros isso ficaria visivel facilmente